proposals/0122-New_rules_for_providing_VRHelpItems_VRHelpTitle.md

```diff
@@ -25,13 +25,16 @@ The proposed mechanism is detailed below:
 	1. construct the `vrHelp` parameter using the data from the list SDL Core internally created.
   	2. construct the `helpPrompt` parameter using the data from the list SDL Core internally created.
   	3. then send these parameters to the HMI via the `SetGlobalProperties` RPC.
-5. If after SDL Core sends the `SetGlobalProperties` RPC, applications sends further AddCommand/DeleteCommand requests then SDL Core shall send `SetGlobalProperties` with the updated parameters.(The full list of parameters needs to be sent and not just the recently added one).
+5. If the timer times out and mobile application does not send SetGlobalProperties and AddCommand requests SDL shall:
+	1. construct the vrHelp parameter using default data from .ini file, e.g. "Unfortunately application has no available commands".
+  	2. construct the helpPrompt parameter using the default data from .ini file, e.g. "Please speak one of the following commands".
+  	3. then send these parameters to the HMI via the `SetGlobalProperties` RPC.
+6. If after SDL Core sends the `SetGlobalProperties` RPC, applications sends further AddCommand/DeleteCommand requests then SDL Core shall send `SetGlobalProperties` with the updated parameters.(The full list of parameters needs to be sent and not just the recently added one).
```

Comment:
Please update the first sentence to " If after SDL Core sends the `SetGlobalProperties` RPC, the application sends further AddCommand/DeleteCommand requests, then SDL Core shall send `SetGlobalProperties` with the updated parameters.